src/edu/washington/escience/myria/operator/KeepAndSortOnMinValue.java

```diff
@@ -182,16 +183,14 @@ public TupleBatch update(final TupleBatch tb) {
     private boolean unique;
 
     @Override
-    public boolean execute(final int index) {
+    public void value(final int index) {
       if (TupleUtils.tupleEquals(inputTB, keyColIndices, row, uniqueTuples, keyColIndices, index)) {
         unique = false;
         Column<?> valueColumn = inputTB.getDataColumns().get(valueColIndex);
         if (shouldReplace(index, valueColumn, row)) {
           uniqueTuples.replace(valueColIndex, index, valueColumn, row);
-          replaced = true;
```

Comment:
@jingjingwang @stechu hmm, is this missing line a bug?

If it is, it doesn't trigger a bug in the tests...